perun/check/factory.py

```diff
@@ -289,10 +314,10 @@ def parse_strategy(strategy):
     return short_strings.get(strategy, strategy)
 
 
-def get_strategies_for(profile):
+def get_strategies_for(profile: Profile) -> Iterable[str]:
     """Retrieves the best strategy for the given profile configuration
 
-    :param ProfileInfo profile: Profile information with configuration tuple
+    :param Profileprofile: Profile information with configuration tuple
```

Comment:
Seems like a small typo here. However, this leads me to a more general issue: Should we still continue using the old type hints in docstrings now that our codebase is mostly type annotated? This seems a bit redundant to me.

Reply:
I kept them there, since it seems that some checkers/IDEs/etc might use them (I think PyCharm---though it might be older version---complained with highlight and it annoyed me. But we might stop use them and slowly remove them.